### PR TITLE
update scripts to process multiple days data

### PR DESCRIPTION
**Pull request by weiqi-tori**

### CODE REVIEW

File: for_003_nrt_rw1_glad_deforestation_alerts/contents/src/__init__.py [hunk 1]
```diff
@@ -192,16 +196,17 @@ def getNewDates(exclude_dates):
...
     # if the current date string is not in the list of dates we already have
     # add the date to the list of new dates to try and fetch 
-    if date.strftime(DATE_FORMAT) not in exclude_dates:
+    while date.strftime(DATE_FORMAT) not in exclude_dates and len(new_dates) < MAX_ASSETS:
         # generate a string from the date
         datestr = date.strftime(DATE_FORMAT)
         # add to list of new dates
         new_dates.append(datestr)
-    else:
+        date = date - datetime.timedelta(days=1)
```
Comment (chrowe) [inline for_003_nrt_rw1_glad_deforestation_alerts/contents/src/__init__.py:208]: What is this doing?

**Review by chrowe — COMMENTED** [review]
What was the issue that this fixed?

**Comment by weiqi-tori** [comment]
> What was the issue that this fixed?

The data source used to update data daily and on time. So we used the script to fetch the latest data (yesterday) every day and it works fine. But recently, the source failed to update data in time. For example, from July 1st to 10th, it only updated data for 7/1, 7/2 and 7/8, and they were also delayed. So the script didn't successfully get the data. I updated the script to search for data from the latest date in our existing data to now, instead of just checking for one day.

File: for_003_nrt_rw1_glad_deforestation_alerts/contents/src/__init__.py (hunk 1 above)
Comment (weiqi-tori) [reply for_003_nrt_rw1_glad_deforestation_alerts/contents/src/__init__.py:208]: Rewind a day in a while loop

**Pull request merged**